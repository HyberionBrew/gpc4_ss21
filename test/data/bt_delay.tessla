in d : Events[Int]
in r : Events[Unit]
def y = delay(d,r)
out y