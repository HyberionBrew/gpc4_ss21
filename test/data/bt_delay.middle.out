8: y = ()
115: y = ()
118: y = ()
162: y = ()
172: y = ()
186: y = ()
274: y = ()
282: y = ()
303: y = ()
353: y = ()
357: y = ()
380: y = ()
414: y = ()
426: y = ()
427: y = ()
441: y = ()
467: y = ()
477: y = ()
480: y = ()
490: y = ()
512: y = ()
542: y = ()
551: y = ()
571: y = ()
577: y = ()
584: y = ()
617: y = ()
618: y = ()
686: y = ()
690: y = ()
732: y = ()
753: y = ()
781: y = ()
811: y = ()
825: y = ()
862: y = ()
876: y = ()
888: y = ()
942: y = ()
961: y = ()
970: y = ()
972: y = ()
1009: y = ()
1014: y = ()
1021: y = ()
